jupyter-docker-stacks.yaml

```diff
@@ -0,0 +1,85 @@
+#nolint:valid-pipeline-git-checkout-commit,valid-pipeline-git-checkout-tag
+package:
+  name: jupyter-docker-stacks
+  version: 7.3.2
+  epoch: 0
+  description: Ready-to-run images containing Jupyter applications
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - py3.12-pip
```

Comment:
Just confirming this doesn't work with 3.13 yet?

Reply:
as of now jupyter images are using python-3.12 version and I intentionally kept it same https://github.com/jupyter/docker-stacks/blob/a6ef176ff3a798af0913260baed10b431066d227/images/docker-stacks-foundation/Dockerfile#L91-L92